app/jobs/reports/identity_verification_report.rb

```diff
@@ -2,17 +2,43 @@
 
 module Reports
   class IdentityVerificationReport < BaseReport
-    REPORT_NAME = 'identity-verification-report'
+    REPORT_NAME = 'identity-verification-report'.freeze
 
     attr_accessor :report_date
 
     def perform(report_date)
       return unless IdentityConfig.store.s3_reports_enabled
       self.report_date = report_date
 
+      emails = IdentityConfig.store.team_ada_email
```

Comment:
The code calls `emails.each`, so it's expected to be an array, but `team_ada_email` is singular, and the values in `application.yml.default` are single emails, not arrays. I think we need to expect a single email and add a mailing list for team Ada, or change the config name to `team_ada_emails` and make the defaults arrays.